src/Chart.jsx

```diff
@@ -351,13 +343,41 @@ class Chart extends Component {
       .attr('fill', 'none')
       .attr('pointer-events', 'all')
       .on('mouseover', () => {
+        g.selectAll('.mouse-per-line').attr('transform', function(d, i) {
```

Comment:
@vasu2707 `transform` is element attribute. Ex: https://developer.mozilla.org/en/docs/Web/SVG/Attribute/transform if you are not returning a valid transform string, the call is misleading. You could use https://github.com/d3/d3-selection#selection_each if you want to do something explicitly.